src/Sulu/Component/phpunit.xml.dist

```diff
@@ -1,9 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit bootstrap="./vendor/autoload.php" colors="true">
+<phpunit bootstrap="./../../../vendor/autoload.php" colors="true">
```

Comment:
This file feels a bit lost here... Why did you have to move it?

Reply:
currently not needed did move it back